test.js

```diff
@@ -426,3 +506,13 @@ const checkPropTypes = (propTypes, props) => {
   console.error = original;
   return errors.join(' | ');
 };
+
+// Return a string representation for the given object.
+const repr = obj => {
+  try {
+    return JSON.stringify(obj);
+  } catch (_) {
+    // The given object must be a recursive data structure.
```

Comment:
I wrote a little more involved version to handle this for the GUI test utils that you may want to repurpose:

https://github.com/juju/juju-gui/blob/develop/jujugui/static/gui/src/app/utils/component-test-utils.js#L82